core/templates/pages/exploration-player-page/layout-directives/learner-local-nav.directive.ts

```diff
@@ -122,14 +127,7 @@ angular.module('oppia').directive('learnerLocalNav', [
                   'pages/exploration-player-page/templates/' +
                   'exploration-successfully-flagged-modal.template.html'),
                 backdrop: true,
-                controller: [
-                  '$scope', '$uibModalInstance',
-                  function($scope, $uibModalInstance) {
-                    $scope.close = function() {
-                      $uibModalInstance.dismiss('cancel');
-                    };
-                  }
-                ]
+                controller: 'ConfirmOrCancelModalController'
               }).result.then(function() {}, function() {
```

Comment:
Do we still need to handle the promise here?

Reply:
Not actually, I've removed it.